src/core/public/chrome/ui/header/header.scss

```diff
@@ -0,0 +1,72 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+.newTopNavHeader {
+  z-index: 1000;
+  padding-left: $euiSizeXS;
+
+  &.shadedTopNavHeader {
+    background-color: $euiColorGhost;
```

Comment:
what does this color? does it make sense to be the same color regardless of light/dark?

Reply:
`shadedTopNavHeader` is not being used but i will investigate.

Reply:
Found it unused and removed.